src/components/layout.js

```diff
@@ -2,24 +2,30 @@ import React from "react"
 import PropTypes from "prop-types"
 import Helmet from "react-helmet"
 
+import Logo from "./logo"
 import Header from "./layout/header"
 import Footer from "./layout/footer"
 
 import styles from "./layout.module.css"
 
-const Layout = ({ children }) => (
+const Layout = ({ children, renderHeaderLogo }) => (
   <div className={styles.root}>
     <Helmet>
       <link rel="stylesheet" href="https://use.typekit.net/bcx8qfd.css" />
     </Helmet>
-    <Header />
+    <Header renderLogo={renderHeaderLogo} />
     <main>{children}</main>
     <Footer />
   </div>
 )
 
 Layout.propTypes = {
   children: PropTypes.node.isRequired,
+  renderHeaderLogo: PropTypes.func,
```

Comment:
Should be required, otherwise :boom:

Reply:
There's a default value for this property on line 28. How would this blow up?

Reply:
Completely drunk. You are right